Improvements

JKrag opened the pull request:
I have made a bunch of improvements. I have tried to do it as multiple individual commits, so it is easier to review. Most of it is formatting / linting and minor rewording, but especially the last 2 commits contain content changes that you should be aware of before running the next training.

JKrag committed:
Renamed exercise files and improved formatting

JKrag committed:
Markdown linting and minor rewording

JKrag committed:
Fix yaml (single * in trigger needs quoting)

JKrag committed:
Simplify and modernize suggested git commands

`git push -d`
is a more understandable way to delete a remote branch.
(i.e. the old `git push origin :<branch>` syntax)

Also I believe that just pushing HEAD is easier to understand
than the creation of a local main.

JKrag committed:
Change suggested naming of pipelines

Name the first pipeline "build" and the integration pipeline "ci".
The previous suggestion seemed to cause some
confusion during trainings.

thedrlambda commented:
The exercises file is just for notes for myself. The reason it is a txt and not an md is that when you import into azure devops it ignores txt files, if it is a md file it will be visible in the menu, and then the participants might get distracted by it. All other changes are 👍

JKrag committed:
Improved description for Managing Work Exercise 4

JKrag committed:
Explain PBI acronym

JKrag commented:
@thedrlambda I have renamed the Exercise file back again (from `.md` to `.txt`) and then cleaned up the history so it was never renamed in the first place. I also added 2 new commits with minor changes based on the experience from the last run.

As you already "soft approved" all the other changes 6 months ago, and we need this tomorrow, I will just merge the branch now. It is Git after all, so we can always revert any changes that "don't work" in the trianing.
